EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -7,10 +7,15 @@
 #include "KeyMotorData.h"
 #include "MotorDetailsData.h"
 #include "DriverControlData.h"
+#include "MotorFaultsData.h"
+#include "BatteryFaultsData.h"
 
 #define KEY_MOTOR_LENGTH (43)
 #define MOTOR_DETAILS_LENGTH (69)
 #define DRIVER_CONTROLS_LENGTH (9)
+#define MOTOR_FAULTS_LENGTH (9)
+#define BATTERY_FAULTS_LENGTH (3)
```

Comment:
I think KEY_MOTOR_LENGTH in here is correct, but the protocol sizes (43)is including the Overhead Byte(1) + Checksum(2) + Frame Delimiter(1) = 47 Bytes.

So yeah you're right, BATTERY_FAULTS_LENGTH should be 7 here, but on the protocol it should be 11 (Just edited it).